includes/class-plugin.php

```diff
@@ -80,13 +80,26 @@ private function __construct() {
             $this->screen = 'settings_page_redis-cache';
         }
 
+        $this->run_migrations();
         $this->add_actions_and_filters();
 
         if ( is_admin() && ! wp_next_scheduled( 'rediscache_discard_metrics' ) ) {
             wp_schedule_event( time(), 'hourly', 'rediscache_discard_metrics' );
         }
     }
 
+    /**
+     * Run migrations
+     *
+     * @return void
+     */
+    public function run_migrations() {
```

Comment:
To simplify things we could just flush the cache every time the plugin is updated.

Alternatively we should avoid polluting the options table with version specific entries - a general `roc_version` storing the last known version would be better. Migrations would only run if the current plugin version is newer.

Reply:
Agreed 👌🏻